src/os/bluestore/ZonedAllocator.cc

```diff
@@ -151,7 +151,7 @@ void ZonedAllocator::dump(std::function<void(uint64_t offset,
 }
 
 void ZonedAllocator::init_from_zone_pointers(
-  std::vector<zone_state_t> &&_zone_states,
+  std::vector<zone_state_t> _zone_states,
```

Comment:
any rationales for that change?

Reply:
this used to get passed directly from ``ZonedFreelistManager::get_zone_states()``.  the caller is done with it at this point so I can std::move, though, so doing that.